### PR TITLE
Add Kim and John to the team page

### PR DESCRIPTION
**Pull request by johnrjj**

### CODE REVIEW

**Comment by chriskalani** [comment]
I wonder why your account isn't attributed to the commit here? 

Screen Shot 2019-12-02 at 4 04 44 PM

**Comment by johnrjj** [comment]
> I wonder why your account isn't attributed to the commit here?
> 
> Screen Shot 2019-12-02 at 4 04 44 PM

Need to set up my git config properly, I'll do that now

**Comment by johnrjj** [comment]
There we go (repushed w/ correct git identity)

File: ts/pages/about/team.tsx [hunk 1]
```diff
@@ -203,6 +203,16 @@ const team: TeamMember[] = [
         name: 'Fulvia Morales',
         title: 'Product Manager',
     },
+    {
+        imageUrl: '/images/team/kimp.jpg',
+        name: 'Kim Persson',
+        title: 'Front-end Engineer',
```
Comment (kimpers) [inline ts/pages/about/team.tsx]: Doesn't really matter but I think I'd prefer a more generic title 😄 
```suggestion
        title: 'Software Engineer',
```

**Review by kimpers — APPROVED** [review]

**Pull request merged**